### PR TITLE
Video link update

### PR DESCRIPTION
**Pull request by dorlinski**
[Problem] Video didn't play
[Solution] Update video source and change video format to mp4.

### CODE REVIEW

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h1>Section Changer</h1>
 It gets to have a virtue of needlessness of changing page and the existance of same header within multiple <span style="font-family: Courier New,Courier,monospace">&lt;section&gt;</span> elements.</p>
 
 <video autoplay muted loop controls>
-	<source src="../../images/SectionChanger.ogv">
+	<source src="https://developer.tizen.org/sites/default/files/documentation/sectiON-CHANGEr.mp4">
```
Comment (lslachciak) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: check file name - it should be lower case everywhere

Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: I changed names to camelCase

**Comment by mijin85cho** [comment]
On current Developer Tizen Site (https://developer.tizen.org), when we upload some media files with capital letters in their name, the developer.tizen.org automatically changes the capital letters to small letters.

So, you can check the valid url for the Developer Tizen with below urls.

 - https://developer.tizen.org/sites/default/files/documentation/sectionchanger.mp4 (working)
 - https://developer.tizen.org/sites/default/files/documentation/sectionChanger.mp4  (not working)

The https://docs.tizen.org, which we will use later, uses the capital/small letters as original file name, so when we move to docs.tizen.org we should use the exact capital/small letter as you requested to check.

However as we are publishing dev guides for Tizen on developer.tizen.org, we should prevent the not displayed videos on developer.tizen.org. That's why I requested to use small letters in the link.
For now please use the small letters, and when we move to docs.tizen.org, please update the small letters to capital letters, as used in the file names.

(or, you can change the file names to use small letters, to reduce the labor in the future.)

**Comment by lslachciak** [comment]
@mijin85cho 
_(or, you can change the file names to use small letters, to reduce the labor in the future.)_
I like this option. Let's avoid duplicaiton of work in future

**Comment by dorlinski** [comment]
Something goes wrong with force-pushed to my fork. I will repair it.

**Pull request merged**